tests/functional/test_install_extras.py

```diff
@@ -44,12 +44,8 @@ def test_no_extras_uninstall(script):
     result = script.pip(
         'install', 'Paste[openid]==1.7.5.1', expect_stderr=True,
     )
-    assert join(script.site_packages, 'paste') in result.files_created, (
-        sorted(result.files_created.keys())
-    )
-    assert join(script.site_packages, 'openid') in result.files_created, (
-        sorted(result.files_created.keys())
-    )
+    result.did_create(join(script.site_packages, 'paste'))
+    result.did_create(join(script.site_packages, 'openid'))
```

Comment:
```suggestion
    result.did_create(join(script.site_packages, 'openid'), sorted(result.files_created.keys()))
```

Reply:
No -- this information is included in `str(result)`.